09_variables_conditionals/solution_student/temp_converter/js/main.js

```diff
@@ -0,0 +1,48 @@
+
+
+function convertFtoC(fTemp) {
+	var fTemp = (fTemp - 32) * (5/9);
+	return fTemp;
+};
+
+$('#fahrenheit_to_celsius').on('click', function() {
+	var typedTemp = $('#temperature').val();
+	typedTemp = parseFloat(typedTemp);
+	var resultTemp = convertFtoC(typedTemp);
```

Comment:
I like how you're storing the return value from the function as a variable here! Great technique! :+1: